src/demos/AirshipModal.tsx

```diff
@@ -20,7 +20,9 @@ interface Props {
 
   // True to have the modal float in the center of the screen,
   // or false for a bottom modal:
-  center?: boolean
+  center?: boolean,
+  // This is to override the set body width for frame
+  styleOverride?: Object,
```

Comment:
I'm not a big fan of generic `style` props, since there is no way to set these up properly without detailed internal knowledge of how the component works. If I decide one day to switch from a margin-based layout to a padding-based layout, or maybe flip around the flex box direction or such, any style props passed in would break - this creates too much API surface area. Instead, I would like a collection of props like `maxHeight`, `backgroundColor`, `borderRadius`, or whatever you specifically want to tweak.

With that said, let's get this in for now, and then as you use it, please keep a note of what you need to tweak so we can just make those changes explicitly supported, and eventually remove `styleOverride`.

Reply:
The proper flow type here is `React.ViewStyle`, not `Object`.